src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppKeys.scala

```diff
@@ -84,6 +84,18 @@ trait SbtReactiveAppKeys {
    */
   val akkaManagementEndpointName = SettingKey[String]("rp-akka-management-endpoint-name")
 
+  /**
+   * Defines packages to install on top of the base alpine image. By default, this includes bash but you can
+   * arbitrarily add to it by using +=, e.g. `alpinePackages += "coreutils"`
+   */
+  val alpinePackages = SettingKey[Seq[String]]("rp-alpine-packages")
```

Comment:
Used to make sure we have bash available. Could be useful for users too, pretty nice syntax to add alpine packages.

Reply:
Okay, I think I'll refactor it into two settings, one that is internal.